### ISSUE TITLE
Effects for brainloss above 200

### ISSUE THREAD

**Issue by kilozombie** [d3athrow/vgstation13#36486]
[featurerequest]
Currently, brain damage has a hard cap: 200 units of brain damage makes you practically unable to open doors, you gibber out random lines constantly, and you have basically no chance of using any items or computers.

However, **sometimes people deserve more brain damage**. It would be funny if there were more effects for 201 and beyond, up to some kind of ridiculous cap. Examples:

- Ever-worsening versions of existing effects
- Randomly using items in hand
- Forgetting people's names and they are all just "Unknown"
- Wiki pages opening on their own
- Imitating things that people just said
- Cycling help/disarm/grab/harm and kick/bite intents randomly
- Say ahelps out loud when you make them
- Drooling bits of brain out your nose
- Chatlog becomes really short
- Immunity to pain

Probably for the best if you get massively diminishing returns past 200 when using something like impedrezene.

**Comment by nervere** [comment]
I disagree that OOC stuff like opening wiki pages/speaking ahelps would be good/fun changes.

**Comment by SECBATON-GRIFFON** [comment]
that third one is a t-thanks BYOND issue i believe, no way to change the name of how an object is perceived to someone cause of right click menus and etc

**Comment by kilozombie** [comment]
> that third one is a t-thanks BYOND issue i believe, no way to change the name of how an object is perceived to someone cause of right click menus and etc

Could it be edited in the chatlog?

I know that things like the Confusion rune can cause you to see people as things other than themselves. Could this loop in somehow?

**Comment by Eneocho** [comment]
I'd make it so there is still a hard cap for injury/chemical induced brainloss at 200, but add effects to brainloss above 200, so an admin can VV 5000 braindamage and get the extra effects, but these are not reachable normally.

**Comment by Eneocho** [comment]
We need a proc that can "retardify" any sentence spoken, so you can have a "i wana lok at teh monkis"-type sentences. Replace any double letter with just that letter, he with eh, ies with is, etc.

**Comment by kilozombie** [comment]
> We need a proc that can "retardify" any sentence spoken, so you can have a "i wana lok at teh monkis"-type sentences. Replace any double letter with just that letter, he with eh, ies with is, etc.

Out of scope, but this would honestly be better than brain damage lines for normal gameplay

**Comment by Eneocho** [comment]
I can probably shitcode a way too long to be useful proc that does the job. Lemme see.
